e2e/components/Breadcrumb/Breadcrumb-test.avt.e2e.js

```diff
@@ -59,25 +59,6 @@ test.describe('@avt Breadcrumb', () => {
     await expect(page.getByText('Breadcrumb 1')).toBeEnabled();
   });
 
-  test('@avt-keyboard-nav - item without href prop', async ({ page }) => {
-    await visitStory(page, {
-      component: 'Breadcrumb',
-      id: 'components-breadcrumb--default',
-      globals: {
-        theme: 'white',
-      },
-    });
-    await expect(page.getByText('Breadcrumb 3')).toBeVisible();
-    await page.keyboard.press('Tab');
-    await page.keyboard.press('Tab');
-    await page.keyboard.press('Tab');
-    await expect(page.getByText('Breadcrumb 3')).toBeFocused();
-
-    // The Breadcrumb 4 should not be focusable
-    await page.keyboard.press('Tab');
-    await expect(page.getByText('Breadcrumb 4')).not.toBeFocused();
```

Comment:
Could we add this to the `@avt-keyboard-nav with overflow menu keyboard navigation` test below?

After the assertion that Breadcrumb 4 is focused we could `tab`, ensure `Breadcrumb 5` is focused, hit `tab` again, `Breadcrumb 6` should _not_ be focused.

This will make sure that we're covering the `isCurrentPage` functionality from a keyboard navigation perspective. Right now `isCurrentPage` isn't covered by tests anywhere else.